doc/modules/ROOT/pages/nrepl-api/ops.adoc

```diff
@@ -1302,7 +1288,7 @@ Returns::
 
 
 
-=== `log-add-appender`
+=== `cider/log-add-appender`
```

Comment:
I'm guessing you were doing some experiments with this.

Reply:
`cider.nrepl.middleware.log` uses ns-qualified ops, so the doc benefits from this tweak